java/src/main/java/ai/rapids/cudf/HostMemoryBuffer.java

```diff
@@ -536,13 +537,27 @@ public final void copyFromDeviceBuffer(BaseDeviceMemoryBuffer deviceMemoryBuffer
    * @param stream CUDA stream to use
    */
   public final void copyFromDeviceBuffer(BaseDeviceMemoryBuffer deviceMemoryBuffer,
-      Cuda.Stream stream) {
+                                         Cuda.Stream stream) {
     addressOutOfBoundsCheck(address, deviceMemoryBuffer.length, "copy range dest");
     assert !deviceMemoryBuffer.closed;
     Cuda.memcpy(address, deviceMemoryBuffer.address, deviceMemoryBuffer.length,
         CudaMemcpyKind.DEVICE_TO_HOST, stream);
   }
 
+  /**
+   * Copy from a DeviceMemoryBuffer to a HostMemoryBuffer using the specified stream.
+   * The copy is async any may not have completed when this returns.
```

Comment:
Must have missed this on my first pass.
```suggestion
   * The copy is async and may not have completed when this returns.
```